v21.1/restore.md

```diff
@@ -20,7 +20,12 @@ You can restore:
 
 ## Required privileges
 
-Only members of the `admin` role can run `RESTORE`. By default, the `root` user belongs to the `admin` role.
+- Full cluster restores can only be run by members of the `admin` role. By default, the `root` user belongs to the `admin` role.
```

Comment:
~Do we have link to something that describes a "full restore"?  Links to what is the admin role?